### ISSUE TITLE
make `QualityFilterStats` viewable as `qiime2.Metadata`

### ISSUE THREAD

**Issue by gregcaporaso** [qiime2/q2-quality-filter#40]
We could then pass this to `qiime metadata tabulate` and drop the `visualize_stats` visualizer. 

For example:

```bash
qiime quality-filter q-score --i-demux demux.qza --o-filtered-sequences demux-filtered.qza --o-filter-stats demux-filter-stats.qza
qiime metadata tabulate --m-input-file demux-filter-stats.qza --o-visualization demux-filter-stats.qzv
qiime tools view demux-filter-stats.qzv
```

**Comment by wasade** [comment]
:+1:

Would it be possible to integrate the qual stats into metadata? It would be nice to, for instance, easily see if the number of reads dropped due to ambiguous characters for technical effect assessment.

**Comment by thermokarst** [comment]
Closed in #42.
